client/app/smallgroup/smallgroup.controller.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+angular.module('observatory3App')
+  .controller('SmallGroupCtrl', function ($scope, $stateParams, $http, Auth, User, $location, notify) {
+    Auth.getCurrentUser(function(user){
+        $scope.user = user;
+        loadSmallGroup();
+    });
+
+    function loadSmallGroup(callback){
+        callback = callback || function(){};
+        $http.get('/api/smallgroup/' + $scope.user.smallgroup).success(function(smallgroup){
+            $scope.smallgroup = smallgroup;
+            callback(smallgroup);
+        });
+        $http.get('/api/smallgroup/' + $scope.user.smallgroup + '/members').success(function(members){
+            $scope.members = members;
+        });
+    }
+
+    $scope.createSmallGroup = function(){
+        $http.post("/api/smallgroup/").success(function(){
+            window.location.reload();
+            notify("Success!");
+        });
+    };
+    $scope.addUser = function(user){
+        $http.get("/api/users/search", {
```

Comment:
The search is currently case sensitive  - this produces some unfriendly behavior.